revista-brasileira-de-ciencia-do-solo.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="never" default-locale="pt-BR">
   <info>
-    <title>Revista Brasileira de Ciência do Solo (Portuguese - Brazil)</title>
+    <title>Revista Brasileira de Ciência do Solo </title>
```

Comment:
@FSHigashikawa, the title should remain "Revista Brasileira de Ciência do Solo (Portuguese - Brazil)".